lms/djangoapps/instructor_task/api.py

```diff
@@ -279,7 +279,10 @@ def submit_bulk_course_email(request, course_key, email_id):
     # We also pull out the targets argument here, so that is displayed in
     # the InstructorTask status.
     email_obj = CourseEmail.objects.get(id=email_id)
-    targets = [target.target_type for target in email_obj.targets.all()]
+    targets = [
+        target.short_display()
+        for target in email_obj.targets.all()
+    ]
```

Comment:
@adampalay This is where we're setting the `task_input` for email tasks, it looks like it can grow without bound (unlimited number of cohorts, select all of them as targets).

This should not be an issue in prod currently, as the cohort options are not yet available.

This data is only needed for display use (see screenshot) The actual values used by the task are stored in the database, keyed on `email_obj`, so I'll just set some sort of limit and truncate the `targets` display.

<img width="1136" alt="screen shot 2016-06-03 at 10 52 41 am" src="https://cloud.githubusercontent.com/assets/7373924/15782708/55214686-2979-11e6-838f-456a813134f1.png">